<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(spread:pad, x1:0.493074, y1:0.818182, x2:0.487685, y2:0.153409, stop:0 rgba(240, 77, 21, 255), stop:0.881773 rgba(244, 131, 0, 255))</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="history">
    <widget class="QPushButton" name="reading1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>150</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>210</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>270</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>330</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>390</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading8">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>450</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading9">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>510</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="reading10">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>570</y>
       <width>410</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 5px;
text-align: left;</string>
     </property>
     <property name="text">
      <string>Reading</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/bar.png</normaloff>:/images/bar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="charts">
    <widget class="QPushButton" name="chartButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: white;
border: 1px solid black;
border-left: none;
border-top: none;</string>
     </property>
     <property name="text">
      <string>Chart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="circleButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
color: white;
border: 1px solid black;
border-right: none;
border-left: none;
border-top: none;
</string>
     </property>
     <property name="text">
      <string>Circle</string>
     </property>
    </widget>
    <widget class="QLabel" name="chartLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>5</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources.qrc">:/images/current_chart.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="circleLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>5</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources.qrc">:/images/current_chart.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
